src/ramses_tx/parsers.py

```diff
@@ -2170,7 +2170,9 @@ def parser_31da(payload: str, msg: Message) -> PayDictT._31DA:
         **parse_outdoor_temp(payload[26:30]),  # 1290?
         **parse_capabilities(payload[30:34]),
         **parse_bypass_position(payload[34:36]),  # 22F7-ish
-        **parse_supply_fan_speed(payload[40:42]),
+        **parse_supply_fan_speed(
+            payload[40:42]
+        ),  # VenturaV1x: 0x00|08|14|28|7E repeated in [40:42]
```

Comment:
Generally, I now want these comments in **tests/tests/parsers/code_*.log**, otherwise the code base gets messy, full of edge-case info.